db/migrate/20220923105541_create_reporting_facility_daily_follow_ups_and_registrations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReportingFacilityDailyFollowUpsAndRegistrations < ActiveRecord::Migration[5.2]
+  # Renaming the view to better match that is reporting on facility state daily dimensions,
+  # i.e. daily breakdowns by diagnosis and gender
```

Comment:
This comment doesn't need to be commited to the repo long term, can just add it in this PR.

Reply:
okay